tests/test_eks/test_eks_constants.py

```diff
@@ -49,6 +51,9 @@
 NODEROLE_ARN_KEY = "nodeRole"
 NODEROLE_ARN_VALUE = "arn:aws:iam::123456789012:role/role_name"
 
+POD_EXECUTION_ROLE_ARN_KEY = "podExecutionRoleArn"
+POD_EXECUTION_ROLE_ARN_VALUE = "arn:aws:iam::123456789012:role/role_name"
```

Comment:
The account ID is centralized, so we should use that over hardcoded values where possible:
```
from moto.core import ACCOUNT_ID
```
(Same for `NODEROLE_ARN_VALUE `, actually - I must have missed that in the last review)